### ISSUE TITLE
Text/font in karchan readability 

### ISSUE THREAD

**Issue by nickicolem** [maartenl/Land-of-Karchan#69]
A couple of players have advised they struggle a bit with the font in karchan on their mobile devices. With mobile devices being more popular to rp on, potentially an option to maybe look to have display/different font options for them just slightly for ease of use.

One person has said the font is too light, so perhaps the option to change the font to pure black.
Another person struggles to read the size of the text on karchan on their phone, other sites are fine so changing the web settings on their phone wouldn't help. So ideally an option in game/ on site to increase the font size would be handy.

**Comment by wastedshame** [comment]
I'm terrible. A player recently told me he was experiencing this issue and sent me screenshots, and I forgot all about it until now. Let me see if I still have them.

**Comment by wastedshame** [comment]
![screenshot_20171001-092251](https://user-images.githubusercontent.com/7554516/31785158-45cfc41a-b4d2-11e7-9976-7a870f2c8e2e.png)
![screenshot_20171001-091826](https://user-images.githubusercontent.com/7554516/31785159-464a0a40-b4d2-11e7-9ad7-30dbd8ba8b6b.png)

**Comment by wastedshame** [comment]
I think that the latter is on one phone, and first one is on his old phone. I know that Ephinie has mentioned before that the font on the admin pages was too light and gave her a headache, but I don't know if she experiences the same issue in game.

**Comment by nickicolem** [comment]
Yeah talking to the people it is phone chrome browser that comes up with the font skinny and light and there isn't a fix for it so ideally if we can get a way then can amend the font then would be fabulous. I tested on my phone too and that is exact the sort of difference I see between android browser and chrome browser. it is vastly different but other sites are fine.
